api_app/resources/strings.py

```diff
@@ -254,3 +254,6 @@
 
 # Value that a sensitive is replaced with in Cosmos
 REDACTED_SENSITIVE_VALUE = "REDACTED"
+
+# Common error message for invalid resource type
+INVALID_RESOURCE_TYPE = "Invalid resourceType. Expected '{}'. Received '{}'."
```

Comment:
I half wondered if printing the "received" string could be used to exfiltrate some information. I'm 99% sure that a string.format in python, using a `{}` placeholder won't allow a variable/environment variable to be printed...but if we wanted to be 100% sure then can we just remove it?

The sender should be able to determine what they sent. The more important thing is that what they send it not expected, and we tell them what is expected.

Reply:
```suggestion
INVALID_RESOURCE_TYPE = "Invalid resourceType. Expected '{}'."
```

So this? I believe I tested it, was a while ago.